### ISSUE TITLE
BERRIES

### ISSUE THREAD

**Issue by Yu-lianka** [Yu-lianka/bird_simulator#2]
Что у нас с ягодами? НЕ прописано что они делают, НЕТ вывода на экран, НЕТ взаимодействия с птичкой.

**Comment by MentorTutor** [comment]
Как дела с ягодами? Они будут в проекте?

**Comment by Yu-lianka** [comment]
(очень надеюсь что будут)
Варианты для реализации ягод:
1 - бустеры: на время дают щит, уменьшают или замедляют птичку, вовсе убирают шипы и тд 
2 - баллы: за сбор ягод + балл (как по мне достаточно скучно)
3 - антиягоды: ускоряют; ядовитые(жестоко), меняют направление птицы???
4 - задавальник: чтобы победить нужно собрать определённое кол-во ягод (возможно разных)
#всё сразу не нужно, но если получится что-то совместить или будут свои идеи - КРУТО!!!

**Comment by miptananas** [comment]
так, ягоды позволяют перейти на следующий уровень, для этого птичка должна съесть ягоду. с течением времени количество очков ( и количество шипов) растут и это становится сделать сложнее, т.е. цель --- как можно быстрее съесть ягоду и перейти на следующий левел. при переходе на новый уровень все боковые шипы сбрасываются

норм?

**Comment by MentorTutor** [comment]
Интересные идеи )
